packages/aurora-save/test/util.test.js

```diff
@@ -0,0 +1,28 @@
+import {
+  getAuroraDirContext,
+  attachMetaData,
+  noteFileName,
+  noteFileExt
+} from "../util.js";
+import fs from "fs";
+import { EditorState } from "draft-js";
+
+test("noteFileExt is actually an extension", () => {
```

Comment:
These test on the file ext seem unecessary.

Reply:
It tests the functionality of the item. Takes 2 seconds to write and 1milisecond to run, but can save tons of time down the road.